spec/ecommerce/resources/adjustment_order_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "spec_helper"
+
+describe Ecommerce::Resources::AdjustmentOrder do
+  describe '.create', vcr: true do
+    let(:params) { { amount: 50.60, description: 'New promotion', valid_from: '2015-05-27', valid_until: '2015-06-27' } }
```

Comment:
Line is too long. [121/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.